dbdialog/src/main/java/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -1456,6 +1469,8 @@ protected void getConnectionSpecificInfo( DatabaseMeta meta ) {
 
       meta.getDatabaseInterface().setNamedCluster( namedClusterList.getValue() );
     }
+
+    this.extraHandler.ifPresent( handler -> handler.getConnectionSpecificInfo( meta ) );
```

Comment:
"get" usually returns something. Where does that return value go in this statement?

Reply:
just used the same method names as DataHandler has, where get/set is used for load/save. there is no return value. should change to load/save?

Reply:
Eh, consistency is good.